algorithms/HPdutility/man/hpdsample.Rd

```diff
@@ -0,0 +1,85 @@
+\name{hpdsample}
+\alias{hpdsample}
+\title{A sampling algorithm for darrays/dframes}
+\description{
+    Samples data from data1 (and when present, data2, maintaining the same
+    order of samples between them) into a new darray/dframe with nSamplePartitions
```

Comment:
I suggest changing "with nSamplePartitions partitions" into "with number of partitions equal to nSamplePartitions"